nx/include/switch/kernel/shmem.h

```diff
@@ -1,18 +1,67 @@
+/**
+ * @file shmem.h
+ * @brief Shared memory object handling
+ * @author plutoo
+ * @copyright libnx Authors
+ * @remark Shared memory differs from transfer memory in the fact that the kernel (as opposed to the user process) allocates and owns its backing memory.
```

Comment:
Don't like referring to the struct as "Shared memory information structure"

Reply:
What should I call the structure then? It can't be just "shared memory object" because that's just the handle.